app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should verify that the command starts with 'cp' and contains exactly three elements. This is required by the task description and checklist: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'